salt/states/user.py

```diff
@@ -409,7 +410,7 @@ def present(name,
 
     # the comma is used to separate field in GECOS, thus resulting into
     # salt adding the end of fullname each time this function is called
-    for gecos_field in ['fullname', 'roomnumber', 'workphone', 'homephone']:
+    for gecos_field in ['fullname', 'roomnumber', 'workphone']: #, 'homephone']:
```

Comment:
Ah, you're right that it's technically valid. I'm still going to remove the `#` however to make it more readable. Thanks.